src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private int radius;
+
+    public Circle() {
+    }
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        this.setColor(color);
+        this.setName(Figures.CIRCLE.name());
+
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * (radius * radius);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: " + getName().toLowerCase()
```

Comment:
```suggestion
        System.out.println("Figure: " + "Circle"
```
Do this instead of a separate variable